Neos.ContentRepository.Core/Classes/Projection/ContentGraph/Subtree.php

```diff
@@ -15,15 +15,7 @@ final class Subtree
     public function __construct(
         public readonly int $level,
         public readonly Node $node,
-        public array $children = []
+        public readonly array $children = []
```

Comment:
```suggestion
        public readonly array $children
```

and we should also explain how the children are indexed, i think numeric? so the @param doc should say `@param array<int, Subtree> $children`